src/lbm/smtc_modem_core/lr1mac/src/smtc_real/src/smtc_real.c

```diff
@@ -2959,6 +2959,23 @@ lr1mac_version_t smtc_real_get_regional_parameters_version( void )
     return version;
 }
 
+status_lorawan_t smtc_real_set_region_sub_band( smtc_real_t* real, uint8_t band )
```

Comment:
Isn't it better to always return `ERRORLORAWAN` when the value is not US915 or AU915?

Reply:
I recommend this code.
```
status_lorawan_t smtc_real_set_region_sub_band( smtc_real_t* real, uint8_t band )
{
    status_lorawan_t status;
    (void)status;

#if defined( REGION_US_915 )    
    if ((status = region_us_915_set_sub_band( real, band )) != OKLORAWAN) return status;
#endif

#if defined( REGION_AU_915 )
    if ((status = region_au_915_set_sub_band( real, band )) != OKLORAWAN) return status;
#endif

    return OKLORAWAN;
}
```

How about that?